src/app/contributors/page.module.css

```
@@ -0,0 +1,249 @@
+  width: 90%;
+  text-decoration-thickness: .4rem;
+  text-decoration-style: solid;
+  text-decoration-color: #000000;
+  border-left: 2px solid #000000;
+  padding-left: 1rem;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  line-height: 1.6;
+  background: #f0f0f0;
+  border-radius: 8px;
+  padding: 0.5rem;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)